modules/mesh-layers/src/simple-mesh-layer/simple-mesh-layer-vertex.glsl.js

```diff
@@ -33,7 +34,7 @@ void main(void) {
   vTexCoord = texCoords;
   cameraPosition = project_uCameraPosition;
   normals_commonspace = project_normal(instanceModelMatrix * normals);
-  vColor = instanceColors;
+  vColor = vec4(colors + instanceColors.rgb, instanceColors.a);
```

Comment:
I think multiplication might be the right choice, rather than addition... I.e completely white instance color should get the vertex color, not stay while. Based on usage that is what I am assuming the ScenegraphLayer / PBR implementation is doing...